ui/components/MeshModelRegistry/MeshModelComponent.js

```diff
@@ -97,6 +97,28 @@ const MeshModelComponent = ({
   const [regi, setRegi] = useState({});
   const [checked, setChecked] = useState(true);
   // const [loading, setLoading] = useState(false);
+  const [windowHeight, setWindowHeight] = useState(window.innerHeight);
+
+  useEffect(() => {
+    const updateWindowHeight = () => {
+      setWindowHeight(window.innerHeight);
+    };
+
+    window.addEventListener('resize', updateWindowHeight);
+
+    return () => {
+      window.removeEventListener('resize', updateWindowHeight);
+    };
+  }, []);
```

Comment:
why do need to use javascript rather than doing with css

Reply:
CSS-only, please.